note about registration order (#26)
GregTechCEu/gtceu-modern-docs: docs/Modpacks/Materials-and-Elements/index.md

Materials are composed of chemical elements and/or other materials.  
Each material can have different items (and blocks), such as ingots, dusts, plates, wires, ores, etc.

## A note about registration

> Order matters when you are registring a new material. If you reference a material by `.components()`, you must either make sure the material is either created before or when on another file loaded first.

Reviewer: Wording is slightly confusing, I'd suggest something like this maybe:

```suggestion
Order matters when you are registering a new material. If you reference a material by `.components()`, you must make sure the other material(s) have been created before the current one.
```
